### PR TITLE
Updated Styled Components to version 6.1.1

### PR DESCRIPTION
**Pull request by HussainAbdi**
update-styled-components: Checkout out the latest version of Styled Components

Problem
See [this issue](https://github.com/AlexanderRichey/styled-react-modal/issues/69).

Solution
Changed the styled-components version in package.json to "^6.1.1". Then I ran `yarn upgrade styled-components@latest`. 

Comments
I believe this is all that needs to be done but let me know if I should be making any other changes! I wasn't sure if there was a command I should be running locally to test or how to see the CodeSandbox in action. EDIT: It seems all checks have passed and I see the CodeSandbox link! How neat.

### CODE REVIEW

**Comment by HussainAbdi** [comment]
The CodeSandbox demo seems to use Styled Components version 5.3.3, wouldn't this need to be updated to a ^6.x.x version in order to sufficiently test this PR? Let me know if there is anything I can do!

File: package.json
Comment (AlexanderRichey) [inline package.json:1]: Can you please bump the version to "3.1.0"?

**Review by AlexanderRichey — CHANGES_REQUESTED** [review]
I changed the Styled Components and React versions in the demo and everything works as expected. Thanks for putting this PR together! I have one small request which is to bump the version to 3.1.0 in the `package.json` file. After that's done, I can merge in the PR and deploy the changes to NPM. 🙂

File: package.json
Comment (HussainAbdi) [reply package.json:1]: Done!

**Comment by HussainAbdi** [comment]
That's great to hear! The version is now bumped.

**Pull request merged**